packages/gestalt/src/Dropdown.js

```diff
@@ -138,6 +138,7 @@ export default function Dropdown({
   const [hoveredItem, setHoveredItem] = useState<number>(0);
 
   const dropdownChildrenArray = Children.toArray(children);
+
```

Comment:
reverting this change will make for cleaner git history, making it clear when material changes were made to this component

Reply:
DONE